htdocs/core/class/commonobject.class.php

```diff
@@ -2119,6 +2121,7 @@ function update_note($note,$suffix='')
 
     	$sql = 'UPDATE '.MAIN_DB_PREFIX.$this->table_element;
     	$sql.= " SET note".$suffix." = ".(!empty($note)?("'".$this->db->escape($note)."'"):"NULL");
+    	$sql.= " ,".($this->table_element == 'adherent'?"fk_user_mod":"fk_user_modif")."=".$user->id;
```

Comment:
There is a lot of tables other than adherent that user fk_user_mod instead of fk_user_modif.
Adding this may generate syntax error for them.
To avoid regression, i moved this PR into develop and tryied to complete it with 3c86507ce6f72ffebb6c664ffe7bce1d5b9726ae